Source/WebCore/platform/cocoa/WebAVPlayerLayerView.mm

```diff
@@ -110,15 +110,16 @@ static void WebAVPlayerLayerView_startRoutingVideoToPictureInPicturePlayerLayerV
     [pipPlayerLayer setVideoGravity:playerLayer.videoGravity];
     [pipPlayerLayer setPlayerController:playerLayer.playerController];
     [pipView addSubview:playerLayerView.videoView];
-    [pipPlayerLayer layoutSublayers];
+    [pipView setNeedsLayout];
+    [pipView layoutIfNeeded];
```

Comment:
Looks like there are some unrelated changes here and below?

Reply:
Whoops! How'd that get in there.